app/mailers/host_mailer.rb

```diff
@@ -47,6 +47,17 @@ def error_state(report)
     group_mail(owners, :subject => (_("Puppet error on %s") % host.to_label))
   end
 
+  def host_built(host)
+    host = Host.find(host)
+    owners = host.owner.recipients_for(:host_built)
+    raise ::Foreman::Exception.new(N_("unable to find recipients")) if owners.empty?
+
+    @host = host
+    set_url
+
+    group_mail(owners, :subject => (_("Host %s is built") % host.to_label))
```

Comment:
Note that this will need rewriting to avoid group_mail, and the recipients stuff will need to move into Host::Managed#built.